### PR TITLE
separate component，make example easyer to understand

### PR DESCRIPTION
**Pull request by liuxuewei**
separate component，make example easyer to understand

### CODE REVIEW

**Comment by facebook-github-bot-8** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

File: examples/flux-todomvc/js/components/TodoItem.react.js [hunk 1]
```diff
@@ -75,6 +76,14 @@ var TodoItem = React.createClass({
...
   },
 
+  _onClick: function() {
+    setTimeout(function () {
+        if(!this.state.isEditing){
+            this.refs.toggleCompleted.getDOMNode().click();
+        }
+    }.bind(this), 200);
+  },
+
```
Comment (fisherwebdev) [inline examples/flux-todomvc/js/components/TodoItem.react.js]: Why this?  This isn't making the example simple at all.

File: examples/flux-todomvc/js/actions/TodoActions.js [hunk 2]
```diff
@@ -78,6 +83,7 @@ var TodoActions = {
     AppDispatcher.dispatch({
       actionType: TodoConstants.TODO_DESTROY_COMPLETED
     });
+    console.log(TodoConstants.TODO_DESTROY_COMPLETED);
```
Comment (fisherwebdev) [inline examples/flux-todomvc/js/actions/TodoActions.js]: If we do these `console.log()` lines, people are going to think that this is a best practice.  I think we should let people put in their own `console.log()`.

File: examples/flux-todomvc/js/components/TodoApp.react.js [hunk 3]
```diff
@@ -14,7 +14,7 @@
 
 var Footer = require('./Footer.react');
 var Header = require('./Header.react');
-var MainSection = require('./MainSection.react');
+var TodoList = require('./TodoList.react');
```
Comment (fisherwebdev) [inline examples/flux-todomvc/js/components/TodoApp.react.js]: (y) Yeah, this is probably better.

File: examples/flux-todomvc/js/components/TodoItem.react.js (hunk 1 above)
Comment (liuxuewei) [inline examples/flux-todomvc/js/components/TodoItem.react.js]: just make TODOList item easier to select

**Comment by kyldvs** [comment]
Thanks for the PR, we recently updated the examples so this is no longer relevant.

https://github.com/facebook/flux/tree/master/examples

**Pull request closed without merging**